docs/site/themes/template/layouts/index.redirects

```diff
@@ -1,3 +1,8 @@
 {{ $latest := (cond (.Site.Params.docs_versioning) .Site.Params.docs_latest "") }}
-/docs                          /docs/{{ $latest }}     301!
-/docs/latest                   /docs/{{ $latest }}
+/docs                                   /docs/{{ $latest }}     301!
+/docs/latest                            /docs/{{ $latest }}
+/docs/package-management                /docs/{{ $latest }}/package-management
```

Comment:
Is this a case where we won't be able to fully see this working until merged? I notice this one is one of the failing links in the `urllint` job:

```
File Path: /home/runner/work/community-edition/community-edition/docs/designs/2967-package-image-vulnerability-scanning.md
URL: https://tanzucommunityedition.io/docs/package-management/
Url Position: 165 : 27
Message: HTTP Status Code: 404
```

Reply:
I think the linter uses the real URL, not the preview URL

Reply:
Oh right, this just sees the link and tries to hit the existing server. Makes sense!